packages/upgrade/test/common/downgrade_component_adapter_spec.ts

```diff
@@ -177,26 +178,28 @@ withEachNg1Version(() => {
       beforeEach(() => registry.unregisterAllApplications());
       afterEach(() => registry.unregisterAllApplications());
 
-      it('should add testabilities hook when creating components', () => {
-
-        let registry = TestBed.get(TestabilityRegistry);
-        adapter.createComponent([]);
-        expect(registry.getAllTestabilities().length).toEqual(1);
-
-        adapter = getAdaptor();  // get a new adaptor to creat a new component
-        adapter.createComponent([]);
-        expect(registry.getAllTestabilities().length).toEqual(2);
-      });
-
-      it('should remove the testability hook when destroy a component', () => {
-        const registry = TestBed.get(TestabilityRegistry);
-        expect(registry.getAllTestabilities().length).toEqual(0);
-        adapter.createComponent([]);
-        expect(registry.getAllTestabilities().length).toEqual(1);
-        adapter.registerCleanup();
-        element.remove !();
-        expect(registry.getAllTestabilities().length).toEqual(0);
-      });
+      fixmeIvy('Runtime compiler is not loaded') &&
```

Comment:
Would you mind prefixing these root causes with the Jira ticket number? e.g. 
```suggestion
      fixmeIvy('FW-561: Runtime compiler is not loaded') &&
```

Reply:
Done